src/App.tsx

```diff
@@ -263,7 +267,22 @@ export const App = ({
 		) {
 			onExpanded(window.performance.now() - expandedStartTime);
 		}
-	}, [isExpanded]);
+
+		if (isExpanded) {
+			// We want react to complete the current work and render, without trying to batch this update
+			// before resetting the number of comments
+			// to the total comment amount.
+			// This allows a quick render of minimal comments and then immediately begin rendering
+			// the remaining comments.
+			setLoadingMore(true);
```

Comment:
We could move this to immediatly after we have fetched, where we're setting `loading` to false - might make more sense there?